lib/phoneNumber.js

```diff
@@ -0,0 +1,61 @@
+var PhoneNumber = function (client) {
+	/**
+		 * Gets a list of all your phone numbers.
+		 * @param {Object} params Query parameters for sorting the results
+		 * @param {String} [params.page=0] Used for pagination to indicate the page
```

Comment:
Please don't document that page parameter. This is deprecated and should not be used. the 'next' header link should be used instead